File: 23-02-2026/biathlon.py
def calculate_penalty_distance(rounds):
    return ((len(rounds) * 5) - sum(rounds)) * 150



print(calculate_penalty_distance([4, 3, 0, 3])); # 1500
print(calculate_penalty_distance([4, 5, 3, 5])); # 450
print(calculate_penalty_distance([5, 5])); # 0
print(calculate_penalty_distance([4, 4])); # 300

File: 23-02-2026/biathlon.js
function calculatePenaltyDistance(rounds) {
  return rounds.reduce((acc, cur) => acc + (5 - cur) * 150, 0);
}

console.log(calculatePenaltyDistance([4, 3, 0, 3])); // 1500
console.log(calculatePenaltyDistance([4, 5, 3, 5])); // 450
console.log(calculatePenaltyDistance([5, 5])); // 0
console.log(calculatePenaltyDistance([4, 4])); // 300
